mqtt/.gitignore

```diff
@@ -1 +1,2 @@
 state*.dat
+unit-tests/
```

Comment:
This is where the rocksdb unit tests dump their state. It isn't getting cleaned up after the test. Should we have a step to explicitly remove this folder after the tests run?

Reply:
yeah, probably it is a good idea. take a look at the https://docs.rs/tempfile/3.1.0/tempfile/